cookbooks/tmate_remote_tmux/recipes/libssh.rb

```diff
@@ -0,0 +1,43 @@
+package "build-essential"
+package "cmake"
+
+libssh_src_dir = "/usr/src/libssh-git"
+execute "install-libssh" do
+  cwd libssh_src_dir
+  command "mkdir -p build && cd build && cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr -DWITH_EXAMPLES=OFF -DWITH_SFTP=OFF .. && make && make install"
+  action :nothing
+end
+
+git libssh_src_dir do
+  repository "https://github.com/nviennot/libssh.git"
+  revision 'v0-7'
+  action :sync
+  notifies :run, "execute[install-libssh]", :immediately
+end
+
+
+# Using the git version for latency metrics instead of the official version.
+
+# libssh_src_dir = "/usr/src/libssh"
```

Comment:
once libssh accepts my pull request (hopefully), eventually we'll go back to the standard releases (.tar.gz instead of git), so I wanted to leave that code in, but it can go away :)